pyext/src/analysis_trajectories.py

```diff
@@ -916,7 +984,7 @@ def plot_hdbscan_clustering(self, S_comb_sel, selected_scores):
             
             
         pl.tight_layout(pad=1.2, w_pad=1.5, h_pad=2.5)
-        fig.savefig(os.path.join(self.analysis_dir,'plot_clustering_scores.png')) 
+        fig.savefig(os.path.join(self.analysis_dir,'plot_clustering_scores'+'.'+self.plot_fmt)) 
```

Comment:
Beware that this plot is set to png because otherwise (i.e. if it is a pdf or other vector file) it is too big. An alternative is to plot only a fraction of the points.

Reply:
Makes sense. 
"plot_clustering_scores" suffix changed back to ".png" in commit 871884ee953b0ce282d98cb31373d20bc0b9218e